build.sbt

```diff
@@ -66,4 +69,15 @@ lazy val ldap = project
   )
   .dependsOn(core)
 
-parallelExecution in Test := false
+lazy val docs = project
+  .in(file("toctoc-docs"))
+  .settings(
+    skip.in(publish) := true,
+    moduleName := "toctoc-docs",
+    mdocVariables := Map(
+      "VERSION" -> version.value,
+      "STABLE_VERSION" -> version.value.replaceFirst("\\+.*", ""),
```

Comment:
What does this regex do?

Reply:
it converts `0.2.0+4-c9659154+20190513-1834-SNAPSHOT` to `2.0.0`